internals/overlord/stateengine.go

```diff
@@ -15,16 +15,24 @@
 package overlord
 
 import (
-	"fmt"
+	"errors"
+	"strings"
 	"sync"
 
 	"github.com/canonical/pebble/internals/logger"
 	"github.com/canonical/pebble/internals/overlord/state"
 )
 
+var ErrStateEngineStopped = errors.New("state engine already stopped")
```

Comment:
It seems like this doesn't need to be exported. Could it be `errStateEngineStopped`?